cmd/server/main.go

```diff
@@ -20,6 +20,7 @@ import (
 
 var k8client *kubernetes.Clientset
 var pushClient *upsClient
+var upsUrlData *UpsURLData
```

Comment:
Sorry for only thinking about this now but maybe we don't need an extra struct and can instead store the URL in the already existing pushClient struct: https://github.com/aerogear/ups-config-operator/blob/master/cmd/server/upsClient.go#L12 

This also contains the application id (in config) that you need to create the final URL which looks like this:
https://<baseUrl>/#/app/<push application id>/variants

Reply:
np - I will update this now.